core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java

```diff
@@ -48,12 +50,12 @@ public ClickstreamImporter(Properties props, ESDriver es, SparkDriver spark) {
   public void addClickStreamMapping() {
     XContentBuilder Mapping;
     try {
-      Mapping = jsonBuilder().startObject().startObject(props.getProperty("clickstreamMatrixType")).startObject("properties").startObject("query").field("type", "string")
+      Mapping = jsonBuilder().startObject().startObject(props.getProperty(MudrodConstants.CLICK_STREAM_MATRIX_TYPE)).startObject("properties").startObject("query").field("type", "string")
           .field("index", "not_analyzed").endObject().startObject("dataID").field("type", "string").field("index", "not_analyzed").endObject()
 
           .endObject().endObject().endObject();
 
-      es.getClient().admin().indices().preparePutMapping(props.getProperty("indexName")).setType(props.getProperty("clickstreamMatrixType")).setSource(Mapping).execute().actionGet();
+      es.getClient().admin().indices().preparePutMapping(props.getProperty("indexName")).setType(props.getProperty(MudrodConstants.CLICK_STREAM_MATRIX_TYPE)).setSource(Mapping).execute().actionGet();
```

Comment:
we should change props.getProperty("indexName")  to props.getProperty(MudrodConstants.INDEX_NAME) in this line